peachjam/views/judgment.py

```diff
@@ -15,7 +15,7 @@ def get_context_data(self, **kwargs):
 
         court_classes = (
             CourtClass.objects.select_related("courts")
-            .order_by("name", "courts__name")
+            .order_by("courts__order", "name", "courts__name")
```

Comment:
I don't think this makes sense. Once the court classes also have an `order` field added, then this can be adjusted to take that into account.